scripts/calculator.js

```diff
@@ -21,12 +21,31 @@ const addNumber = (num) => {
     currentStep = 'firstNumber'
   } else {
     currentStep = 'secondNumber'
-  }
+  } 
 
   state[currentStep] += num
   updateScreen(state[currentStep])
 }
 
+const addDecimal = () => {
+  let currentStep
+
+  // if the 'operator' has been set we know we're dealing with the second number
+  if (state['operator'] === '') {
+    currentStep = 'firstNumber'
+  } else {
+    currentStep = 'secondNumber'
+  } 
+
+  if (state[currentStep].indexOf('.') == -1) {
+    (state[currentStep]) += ('.')
```

Comment:
The "(" ")" brackets here are unnecessary.

Reply:
Im not sure what this is actually doing, i thought currentStep could represent two different numbers. why would this lock you out of a decimal point on the second number?

Reply:
I also am not sure of which "('')" to remove, indexOf changes without them i think.

Reply:
1. User adds a '.' to the first number.
2. User tries to add a second '.' to the first number -> button is disabled.
3. User moves to the second number. They try to add a '.' to the second number -> the button is still disabled.

Reply:
As far as "(" ")" is concerned, I'm referring to line 41:
`(state[currentStep]) += ('.')`

Reply:
is it because it there is nothing re-enabling it?